modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -60,12 +60,14 @@ The following snippet configures an Ethernet interface that uses a dynamic IP ad
 ...
```

Comment:
You changed the example to a static one, but left the intro in line 56 with "The following snippet.... that uses a *dynamic* IP address".

Reply:
Oops. What I was saying and what I listed above are different. My apologies!

Reply:
Updated the content.